[Vitamins Included] Added found design flaws in rev 2.0 (#55)
qmk/qmk.fm: _i18n/en/keyboards/vitamins_included.md

![Bottom render](images/bot.png)

### Top side

> ## NOTE: Rev 2.0 has an error in the labeling of the LED_O jumpers on the right half. They are swapped. Just the lables are swapped. Functionally they're identical.

Reviewer: ```suggestion
#### NOTE: Rev 2.0 has an error in the labeling of the `LED_O` jumpers on the right half. The labels are swapped, but are functionally identical to the previous revision.
```
